apps/file/utils.py

```diff
@@ -28,6 +28,8 @@ def process_excel_file(dataset: Dataset):
                 "preview_data": preview_data or {},
                 "has_errored": err is not None,
                 "error": err or None,
+                # Setting this for now, might change later
+                "is_added_to_workspace": True,
```

Comment:
This should by default be `False`. I do not understand why it is set `True` here.

Reply:
Removed.